src/components/Header/Header.jsx

```diff
@@ -1,6 +1,12 @@
-// import the React library here
-// import a css file containig article styles
+import React from 'react';
+import './Header.css';
+import Novigation from '../Novigation/Novigation';
 
-// Create a Header function returning the HTML of header block
+const Header = () => (
+  <header className="header">
+    <h1 className="header__title">Site Name</h1>
+    <Novigation />
```

Comment:
```suggestion
    <Navigation />
```
rename your component